ontotext-yasgui-web-component/src/models/external-yasgui-configuration.ts

```diff
@@ -248,6 +248,16 @@ export interface ExternalYasguiConfiguration {
   onQueryAborted?: (req) => Promise<void>;
 
   yasrToolbarPlugins?: YasrToolbarPlugin[];
+
+  /**
+   * If this function is present, then it will be called on every one result cell.
+   * @param binding - binding value that will be shown into a cell.
+   * @param prefixes - Object with uris and their corresponding prefixes.
+   */
+  // @ts-ignore
+  getCellContent: (binding: Parser.BindingValue, prefixes?: { [label: string]: string }) => string;
```

Comment:
What's wrong with the `Record<string, string>`?

Reply:
Actually there is a type "Prefixes" exported from Yasr, I will use it.

Reply:
Done